lib/codegen/fromcto/golang/golangvisitor.js

```diff
@@ -283,6 +289,16 @@ class GoLangVisitor {
     toGoPackageName(namespace) {
         return namespace.replace(/@/g, '_').replace(/\./g, '_');
     }
+
+    /**
+     * Check if field is a Map Declaration
+     * @param {Field} field - the field being visited
+     * @return {boolean} true if field is a Map Declaration
+     * @private
+     */
+    isMap(field) {
+        return (Object.keys(field).length > 0 && ModelUtil.isMap?.(field));
```

Comment:
Why do we need `Object.keys(field).length > 0`? 

Should that check be pushed down into ModelUtil?

Reply:
We don't actually need it, this was a copy pasta from the csharp visitor extension I wrote. I don't recall why the field length was necessary. Now using the ModelUtil function in its place.